src/hydromt_fiat/fiat.py

```diff
@@ -194,11 +194,11 @@ def setup_hazard(
         ----------
         hazard_fnames : list[Path | str] | Path | str
             Path(s) to the hazard file(s) or name(s) of the data catalog entries.
+        hazard_type : str, optional
+            Type of hazard, by default "flooding".
```

Comment:
If possible I'd like to see either which hazards are possible, or a link to a place that documents it